tests/src/harness/mod.rs

```diff
@@ -65,15 +57,15 @@ impl<'a> Harness<'a> {
     fn new(
         root: &Path,
         shell: &'a Shell,
-        jumper: &'a Autojumper,
+        jumper: &'a dyn Autojumper,
         preinit: Option<&str>,
-        cgroup: bool,
     ) -> Self {
         let ps1 = &format!("=={}=={}==>", shell.name(), jumper.to_str());
         shell.setup(&root, jumper, ps1, preinit.unwrap_or(""));
 
+        let use_cgroup = std::env::var("PAZI_TEST_CGROUP") == Ok("true".to_string());
```

Comment:
There's definitely a better way to plumb this option in, and I'll probably go back to do that later, but for now this seems expedient.